.env.docker

```diff
@@ -14,9 +17,11 @@ CX_AUTH_REDIRECT_URL=http://localhost:8080/callback
 CX_AUTH_ROLE_MAPPINGS=create_wallets:add_wallets,view_wallets:view_wallets,update_wallets:update_wallets,delete_wallets:delete_wallets,view_wallet:view_wallet,update_wallet:update_wallet
 CX_AUTH_RESOURCE_ID=ManagedIdentityWallets
 
-ACAPY_API_ADMIN_URL=http://localhost:11000
 ACAPY_NETWORK_IDENTIFIER=local:test
-ACAPY_ADMIN_API_KEY=Hj23iQUsstG!dde
+ACAPY_BASE_WALLET_API_ADMIN_URL=http://localhost:11000
+ACAPY_BASE_WALLET_ADMIN_API_KEY=Hj23iQUsstGXdde
+ACAPY_API_ADMIN_URL=http://localhost:11003
+ACAPY_ADMIN_API_KEY=Hj23iQUsstGXdde
 
 BPDM_DATAPOOL_URL=https://catenax-bpdm-int.demo.catena-x.net
```

Comment:
Does this file contain sensitive data in Line 5, 8, 22 and 24? If yes, you should consider these data as compromised and needs to be revoked and set to new values! Hard coding secrets in public repositories is an absolute no-go!

Also hard coded URLs to consortia URLs seems to be wrong (Line 26)

Reply:
Thanks. No, it does not contain any sensitive data. Those values (and the values in env.example) are given only as an example and are not used in any environment.

Reply:
Is it somewhere in documentation mentioned to change the content of these example key contents? Readme.md explains the purpose, but not to replace the content.

Also If you check Catena-X Pre-Prod Environment, you can find the Keys in "production": 

```shell
[~/projects/cat-x]$ k config current-context
cx-pre-prod-aks
[~/projects/cat-x]$ k -n product-managed-identity-wallets \
    get secrets catenax-managed-identity-wallets-acapy-secrets -o yaml | \
    yq '.data.acapy-admin-api-key' | base64 -d
Hj23iQUsstG.dde
[~/projects/cat-x]$ k -n product-managed-identity-wallets \
    exec pods/catenax-managed-identity-wallets-67f7975cdf-7lcrk \
    --container catenax-managed-identity-wallets -- \
    env | grep -i key
ACAPY_ADMIN_API_KEY=Hj23iQUsstG.dde
```

From my point of view there is a high chance that vulnerable installations will be available with current setup.

Reply:
The values are similar but not equal. But yes, you are right this is not well documented and the given values could influence the chosen API-Key and make it insecure. I changed it in .env.example and .env.docker to "ApiKeySecretPlaceholder..." and we will make sure to improve those things in the future.

Reply:
I changed also the JDBC-URL. I hope everything is okay now and it can be merged. Thanks for your time @carslen